nginx.montagu.conf

```diff
@@ -79,12 +71,17 @@ server {
         proxy_pass http://contrib/;
     }
     location /reports/ {
-        proxy_pass http://report/;
-    }
+        proxy_pass http://orderly_web_web:8888/;
+        proxy_redirect default;
 
+        location "~/reports/(?<name>.+)/(?<version>\d{8}-\d{6}-[0-9a-f]{8})" {
```

Comment:
Does the RE `(?<name>.+)` need to be made less greedy, either by using `(?<name>.+?)` or via `(?<name>[^/]+)` - otherwise I think this section will match `foo/bar/20181010-010203` as well as `foo/20181010-010203` (the latter will definitely work, the former might depend on the regex engine that nginx uses)

Reply:
ah yes you're totally right